src/Microsoft.OpenApi.OData.Reader/Operation/RefPostOperationHandler.cs

```diff
@@ -66,7 +67,7 @@ protected override void SetResponses(OpenApiOperation operation)
         {
             OpenApiSchema schema = new OpenApiSchema
             {
-                Type = "String" // What to return?
+                Type = "object" // What to return?
```

Comment:
```suggestion
                Type = "object",
                AdditionalProperties = new OpenApiSchema { Type = "object" }
```

Reply:
This should only apply to request objects via `SetRequestBody`.

Reply:
@peombwa  We shouldn't really do this funky AdditionalProperties in the OpenAPI library as it is an AutoREST issue. But we can fix this later.